Lib/idlelib/idle_test/test_query.py

```diff
@@ -1,4 +1,4 @@
-"""Test query, coverage 93%).
+"""Test query, (coverage 93%).
```

Comment:
I have not used () in any of the other test files.
```suggestion
"""Test query, coverage 93%.
```